oak/server/rust/oak_runtime/src/node/logger.rs

```diff
@@ -37,7 +38,12 @@ impl LogNode {
 impl super::Node for LogNode {
     /// Main execution loop for the logging pseudo-Node just waits for incoming
     /// `LogMessage`s and outputs them.
-    fn run(self: Box<Self>, runtime: RuntimeProxy, handle: oak_abi::Handle) {
+    fn run(
+        self: Box<Self>,
+        runtime: RuntimeProxy,
+        handle: oak_abi::Handle,
+        _: oneshot::Receiver<()>,
```

Comment:
Keep the name of the parameter here for documentation, i.e. `_notify_receiver` (and elsewhere)